src/test/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandlerProviderImplTest.java

```diff
@@ -84,15 +93,17 @@ public void whenTenantDoesNotExist_shouldReturnEmptyOptional() {
     @Test
     public void shouldPatchShippingAddress() {
         CtpPaymentWithCart ctpPaymentWithCart = createCartWithPayment();
-        Payment paypalPlusPayment = paymentMapper.ctpPaymentToPaypalPlus(ctpPaymentWithCart);
-        paypalPlusPayment = executeBlocking(this.paypalPlusFacade.getPaymentService().create(paypalPlusPayment));
-
         PaymentHandler paymentHandler = paymentHandlerProvider.getPaymentHandler(MAIN_TEST_TENANT_NAME).get();
-        PaymentHandleResponse paymentHandleResult = paymentHandler.patchAddress(ctpPaymentWithCart.getCart(), paypalPlusPayment.getId());
+        paymentHandler.createPayment(ctpPaymentWithCart.getPayment().getId());
+
+        Cart cartWithExpansion = executeBlocking(this.ctpFacade.getCartService().getByPaymentId(ctpPaymentWithCart.getPayment().getId(),
+                "paymentInfo.payments[*]")).get();
```

Comment:
as mentioned before this string could be used as a util constant? not hardcoded every time.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/49/commits/f7a43647710700ac50318bce79681e0d1d9179cc